pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -183,10 +185,15 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
 		PullSecret:   installConfig.PullSecret,
 		ReleaseImage: releaseImage,
 		EtcdCluster:  strings.Join(etcdEndpoints, ","),
+		Proxy:        installConfig.Proxy,
 	}, nil
 }
 
 func (a *Bootstrap) addStorageFiles(base string, uri string, templateData *bootstrapTemplateData) (err error) {
+	proxy := map[string]struct{}{
```

Comment:
is it possible to move this to the function calling this function ie in the Generate for bootstrap ignition asset...?